nevergrad/optimization/test_optimizerlib.py

```diff
@@ -70,7 +70,7 @@ def check_optimizer(
             warnings.filterwarnings("ignore", category=FinishedUnderlyingOptimizerWarning)
             # now optimize :)
             candidate = optimizer.minimize(fitness)
-        if verify_value and "chain" not in str(optimizer_cls):
+        if verify_value and "chain" not in str(optimizer_cls) and "EMNA" not in str(optimizer_cls):
```

Comment:
> I have deactivated the check_optimizer test (for emna algorithms) as this kind of algorithms needs a num_workers >> 1

Can you elaborate on this? I don't understand how that is a problem. If num_workers is 1 the algorithms can always mock that the `ask` are played in parallel.

Reply:
If I understood the test correctly, the result is that the population size of emna (with such a parametrization) will be be too small ; and sigma is computed as the standard deviation of the mu (roughly lambda/4) best parents. Maybe I should add a warning in the algorithm to prevent its use with a too small popsize.

Reply:
```python
self.llambda = 4 * self.dimension
if num_workers is not None:
    self.llambda = max(self.llambda, num_workers)
```
This sets the population size to **at least** `num_workers` so `num_workers=1` should not matter

Reply:
Can it be removed or the test is failing for some variants? if so which one? We'd need to figure out why, because num_workers=1 shouldnt be a problem